### PR TITLE
Lps 198333 - Option to disable redirect URL message

### PR DESCRIPTION
**Pull request by manuelRives**
https://liferay.atlassian.net/browse/LPS-198333

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java [hunk 1]
```diff
@@ -967,7 +979,7 @@ private String _normalizeFriendlyURL(String friendlyURL) {
 
 	private Locale _setAlternativeLayoutFriendlyURL(
 		HttpServletRequest httpServletRequest, Layout layout,
-		String friendlyURL, SiteFriendlyURL siteFriendlyURL) {
+		String friendlyURL, SiteFriendlyURL siteFriendlyURL, long companyId) {
```
Comment (adolfopa) [inline modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java:982]: Can you put the companyId as the first argument?

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java [hunk 2]
```diff
@@ -929,6 +929,18 @@ private boolean _isPermanentRedirect(long companyId) {
 		return false;
 	}
 
+	private boolean _isShowRedirectAlternativeFriendlyURLMessage(
```
Comment (adolfopa) [inline modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java:932]: Why not call this `alternativeLayoutFriendlyURLMessage` for consistency with the attribute name that was used originally? Use that also for other methods/lang keys.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java (hunk 2 above)
Comment (manuelRives) [reply modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java]: Done

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java (hunk 1 above)
Comment (manuelRives) [reply modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java]: Done

**Review by marcogalluzzi — APPROVED** [review]
Well done!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
@manuelRives Can you fix the conflicts and forward it when done? Thanks!

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
Failures are unrelated.

**Comment by WYZ-Issac** [comment]
ci:forward:force

**Pull request closed without merging**